frontend/controller/backend.js

```diff
@@ -31,6 +31,9 @@ function signJSON (json, keypair) {
   }, json)
 }
 
+console.info('GroupIncome version:', process.env.GI_VERSION)
+sbp('okTurtles.data/set', 'GI_VERSION', process.env.GI_VERSION)
```

Comment:
I do not see a reason to create this, we should be able to just use `process.env.GI_VERSION` right?

Reply:
That's right. I added this to allow access to the version from within the console by doing `sbp('okTurtles.data/get', 'GI_VERSION')`. I've used it a few times while debugging this PR, but it could also be removed.

Reply:
Yeah it's OK to have stuff that isn't necessary while debugging, but before the code is merged such things should be removed to keep the codebase as clean as we can. It's a constant battle to keep the codebase clean.